jenkins/e2e-image/Dockerfile

```diff
@@ -45,10 +45,11 @@ ENV FAIL_ON_GCP_RESOURCE_LEAK=true \
 
 ADD ["e2e-runner.sh", \
     "kops-e2e-runner.sh", \
+    "https://raw.githubusercontent.com/kubernetes/kubernetes/master/cluster/get-kube.sh", \
```

Comment:
I think it's what you get when you do a `curl https://get.k8s.io/ | bash`. Not test-infra specific. I think it's kinda gross to include it in the `e2e-image`, but I don't know a better way to do this.